modules/light-clients/09-localhost/light_client_module.go

```diff
@@ -36,21 +51,11 @@ func (l *LightClientModule) RegisterStoreProvider(storeProvider exported.ClientS
 	l.storeProvider = storeProvider
 }
 
-// Initialize ensures that initial consensus state for localhost is nil.
+// Initialize returns an error because it is stateless
 //
 // CONTRACT: clientID is validated in 02-client router, thus clientID is assumed here to be 09-localhost.
-func (l LightClientModule) Initialize(ctx sdk.Context, clientID string, _, consensusStateBz []byte) error {
-	if len(consensusStateBz) != 0 {
-		return errorsmod.Wrap(clienttypes.ErrInvalidConsensus, "initial consensus state for localhost must be nil.")
-	}
-
-	clientState := ClientState{
-		LatestHeight: clienttypes.GetSelfHeight(ctx),
-	}
-
-	clientStore := l.storeProvider.ClientStore(ctx, exported.LocalhostClientID)
-	clientStore.Set(host.ClientStateKey(), clienttypes.MustMarshalClientState(l.cdc, &clientState))
-	return nil
+func (LightClientModule) Initialize(ctx sdk.Context, clientID string, clientState, consensusStateBz []byte) error {
```

Comment:
unsure if its necessary but could `_` unused args, we had a linter complain at this at some point before (I'm not sure we still use it tho)

Reply:
I will try and report back!

Reply:
looks to be fine. Will update all args unused in the module to be discarded